### PR TITLE
Environment Variable Tile Server

### PR DESCRIPTION
**Pull request by BryonLewis**
Switches the front-end to using an environment variable for the tile server instead of the hardcoded URLS.

Note this will need to be set during the build process so that the production build has the proper variable.
I made it default to the new URL if it can't find the env anyways.
ALSO:
```
VITE_TILE_SERVER_URL="https://basemap.kitware.watch"
```
Will need to be added to the .env file.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@ services:
   # RD-WATCH
   rdwatch:
     image: ghcr.io/resonantgeodata/rd-watch/rdwatch:latest
+    build:
```
Comment (mvandenburgh) [inline docker-compose.yaml:9]: What is the purpose of this change? It looks like it will change the semantics of the `image` property here (https://stackoverflow.com/a/34316224)

**Review by mvandenburgh — APPROVED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (BryonLewis) [reply docker-compose.yaml:9]: thanks, that was left over from when I was doing local builds.  I'll remove it.
